discovery-infra/tests/base_test.py

```diff
@@ -180,6 +181,20 @@ def _collect_virsh_logs(self, nodes, log_dir_name):
         infra_utils.run_command(f"journalctl --since \"{nodes.setup_time}\" "
                                 f"-u libvirtd -D /run/log/journal >> {libvird_log_path}", shell=True)
 
+    def _collect_journalctl(self, nodes, log_dir_name):
+        logging.info('Collecting journalctl\n')
+        os.makedirs(log_dir_name, exist_ok=True)
+        journal_ctl_path = os.path.join(log_dir_name, "journalctl")
+        os.makedirs(journal_ctl_path, exist_ok=False)
```

Comment:
The first directory is OK to exist but its subdirectory not?
You should expect a raise.. What is the behavior you expect? I would suggest to delete the `journal_ctl_path` dir in case it exists and you don't want it to exists.